app/controllers/ReplacingLeadTrusteeController.scala

```diff
@@ -98,7 +99,7 @@ class ReplacingLeadTrusteeController @Inject()(
     trustees
       .zipWithIndex
       .filter(_._1 match {
-        case trustee: TrusteeIndividual => !trustee.mentalCapacityYesNo.contains(false)
+        case trustee: TrusteeIndividual => !trustee.mentalCapacityYesNo.contains(No)
```

Comment:
I think we need to include Don't know here as well. We are going to allow trustees to be promoted to lead if we've had an **explicit yes** from the user, therefore not showing "no" or "don't know" answers.